chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}},
 creating an authoritative source for the existence and fixed
 characteristics of each of the units that may be surveyed.\sidenote{
-	\url{https://dimewiki.worldbank.org/Unit_of_Observation}}
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}}
 The master dataset indicates how many individuals are eligible for data collection,
-and therefore contains statistical information about the likelihood that each will be chosen.
+and therefore contains statistical information about
+the likelihood that each will be chosen.
 
+% implement uniform-probability random sampling
 The simplest form of random sampling is \textbf{uniform-probability random sampling}.
 This means that every observation in the master dataset
 has an equal probability of being included in the sample.
 The most explicit method of implementing this process
 is to assign random numbers to all your potential observations,
 order them by the number they are assigned,
 and mark as ``sampled'' those with the lowest numbers, up to the desired proportion.
-(In general, we will talk about sampling proportions rather than numbers of observations.
-Sampling specific numbers of observations is complicated and should be avoided,
-because it will make the probability of selection very hard to calculate.)
 There are a number of shortcuts to doing this process,
 but they all use this method as the starting point,
 so you should become familiar with exactly how it works.
-The do-file below provides an example of how to implement uniform-probability sampling in practice. 
+The do-file below provides an example of
+how to implement uniform-probability sampling in practice.
+This code uses a Stata built-in example dataset and is fully reproducible,
+so anyone that runs this code in any version of Stata later than 13.1
+(the version set in this code)
+will get the exact same randomized results.
 
 \codeexample{simple-sample.do}{./code/simple-sample.do}
 
+% what to think about before sampling
 Almost all of the relevant considerations for sampling come from two sources:
 deciding what population, if any, a sample is meant to represent (including subgroups);
-and deciding that different individuals should have different probabilities
+and deciding whether different individuals should have different probabilities
 of being included in the sample.
 These should be determined in advance by the research design,
 since otherwise the sampling process will not be clear,
 and the interpretation of measurements is directly linked to who is included in them.
 Often, data collection can be designed to keep complications to a minimum,
 so long as it is carefully thought through from this perspective.
-Ex post changes to the study scope using a sample drawn for a different purpose
+Ex-post changes to the study scope using a sample drawn for a different purpose
 usually involve tedious calculations of probabilities and should be avoided.
```

Comment:
suggest deleting the whole sentence, or at least the 'should be avoided'

Reply:
Agreed. This is too detailed statistics for this book. Deleted whole sentence here: 26cea3645165d5523b2e263fc0a3177b71fb419c